scripts/macpython-build-wheels.sh

```diff
@@ -64,7 +71,10 @@ pushd ITK-source > /dev/null 2>&1
   ${CMAKE_EXECUTABLE} -DITKPythonPackage_BUILD_PYTHON:PATH=0 \
```

Comment:
The external project `ITK-source` was intended to download ITK sources and not to build other libraries, hence the regression.

Once the release is done, we should revisit this.